backend/director/tools/muapi_video.py

```
@@ -0,0 +1,204 @@
+            "duration": int(duration),
+            "aspect_ratio": config.get("aspect_ratio", "16:9"),
+        }
+
+        try:
+            request_id = self._submit(endpoint, payload)
+            video_url = self._poll(request_id)
+            video_data = requests.get(video_url, timeout=120)
+            video_data.raise_for_status()
+            with open(save_at, "wb") as f:
+                f.write(video_data.content)
+        except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)